missing 7.2.0 language compatibility setting (#3393)
couchbase/docs-server: modules/eventing/pages/eventing-Terminologies.adoc

* *Language compatibility*: The language version of the Eventing Function for backward compatibility.
+

> If the semantics of a language construct change in any given release the “Language compatibility” setting will ensure an older Eventing Function will continue to see the runtime behavior that existed at the time it was authored, until such behavior is deprecated and removed. Note 6.0.0, 6.5.0, 6.6.2, and 7.2.0 are the only currently defined versions and for newly authored Functions the default is the highest compatibility version available, currently 7.2.0.

Reviewer: ```suggestion
If the semantics of a language construct change in any given release, use the *Language compatibility* setting to ensure continued compatibility for Eventing Functions authored using an older version, until the runtime behavior is deprecated and removed. 

Currently, version 6.0.0, 6.5.0, 6.6.2, and 7.2.0 are available. New Eventing Functions will use the highest available version.
```

Author: Looks fine to me
